src/systempolicy/systemPolicy.go

```diff
@@ -1261,8 +1268,11 @@ func GenFileSetForAllPodsInCluster(clusterName string, pods []types.Pod, settype
 		wpfs.FromSource = slog.Source
 		wpfs.SetType = settype
 		labels, err := GetPodLabels(slog.ClusterName, slog.PodName, slog.Namespace, pods)
+		if labels == nil {
```

Comment:
Please explain this line of code.

Reply:
In case of un-orchestrated containers, we won't be receiving any labels, so the labels are derived from the containername

Reply:
Agreed. But before performing such operation please do make sure to check if the log is related to un-orchastrated containers.